src/formatter.rs

```diff
@@ -88,6 +89,32 @@ where
                 serializer.serialize_entry("level", level)?;
             }
 
+            #[cfg(feature = "ansi_logs")]
+            {
+                let level = match *metadata.level() {
+                    tracing::Level::ERROR => "error",
+                    tracing::Level::WARN => "warn",
+                    tracing::Level::INFO => "info",
+                    tracing::Level::DEBUG => "debug",
+                    tracing::Level::TRACE => "trace",
+                };
+
+                if self.with_level {
+                    let level_str = match level {
+                        "error" => nu_ansi_term::Color::Red,
```

Comment:
I think it would be better to match over the level enum here again, that way if a level was added or changed, we'd get a compile time error